.github/workflows/pr-check_markdownlint.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       - name: Setup Node.js environment
         uses: actions/setup-node@v3
         with:
-          node-version: 18
+          node-version-file: ".nvmrc"
```

Comment:
Just did not add `- .nvmrc` to trigger this workflow when node version changed (as we have `markdown-lint.yml` which would lint all the markdown file while this workflow would not works well when there was not changed markdown file)